library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -376,13 +376,15 @@ class Client() {
 
         if (v3Client != null) {
             v3Client.signatureRequest()?.let { signatureRequest ->
-                if (account != null) {
-                    account.sign(signatureRequest.signatureText())?.let {
-                        signatureRequest.addEcdsaSignature(it.rawData)
+                if (!signatureRequest.isReady()) {
```

Comment:
Have you tried removing this `isReady` check. I'm still unconvinced it actually matters with the way you're using `libxmtp`

Reply:
Yep doesn't seem to be needed removed it.